core/src/main/resources/error/error-classes.json

```diff
@@ -25,6 +25,10 @@
   "CONCURRENT_QUERY" : {
     "message" : [ "Another instance of this query was just started by a concurrent session." ]
   },
+  "DATETIME_OVERFLOW" : {
+    "message" : [ "The '%s' function overflows the input '%s' timestamp by %s %s." ],
```

Comment:
can we make it more general? `Datetime operation overflows.` There are so many datetime operations, I think we can omit the input details.

Reply:
I don't think making errors too general is right approach. Error messages should help users to identify and fix issues in their queries.

> I think we can omit the input details.

Could you explain how users should find the problematic row if we don't provide any context as you propose?